bin/install-production-tools.sh

```diff
@@ -0,0 +1,14 @@
+echo "Installing Eepromtool"
+cd /home/user/
+wget https://github.com/shadow-robot/sr-build-tools/raw/$(echo $toolset_branch | sed 's/#/%23/g')/bin/eepromtool
+chmod +x /home/user/eepromtool
+echo "Installing Gedit"
+apt-get -y install gedit
+
+echo "Installing Multiplot"
+apt-get -y install ros-${ros_release_name}-rqt-multiplot
```

Comment:
ros_release_name should be passed as shell script parameter not global env variable